doc/api/intl.md

```diff
@@ -16,6 +16,7 @@ programs. Some of them are:
 - The [WHATWG URL parser][]'s [internationalized domain names][] (IDNs) support
 - [`require('buffer').transcode()`][]
 - More accurate [REPL][] line editing
+- [`util.TextDecoder`][]
```

Comment:
I am not sure? Is the `require('buffer').transcode()` due to the note: "Note that this is a property on the `buffer` module returned by `require('buffer')`, not on the `Buffer` global or a `Buffer` instance"? If not, I will add `require()` for consistency.